client.go

```diff
@@ -1756,6 +1756,8 @@ func (f *File) writeAt(b []byte, off int64) (written int, err error) {
 // Giving a concurrency of less than one will default to the Client’s max concurrency.
 //
 // Otherwise, the given concurrency will be capped by the Client's max concurrency.
+//
+// This method is preferred over calling ReadFrom to guarantee concurrent reads/writes.
```

Comment:
I don’t know if we want to use the word “preferred”, or at least I would want to move the reason/motivation to the front.

That is, “When one needs to guarantee concurrent reads/writes, this method is preferred over ReadFrom.”